### DISCUSSION TITLE
Sticky Menu

### DISCUSSION THREAD

**Discussion by TsinJu** [protz/thunderbird-conversations#2083] [Ideas]
The two areas where the mail address is displayed and where you can show blocked content should be sticky at the top and not disappear when the mail is scrolled. 
Maybe this can be realized in future releases. Thank you very much.

**Comment by Standard8** [comment]
It might be an idea to make the header sticky, changing with each message, but I think it'd be too much to have the notification bar as well - especially in the three pane view which has limited height.
